### ISSUE TITLE
Question: How to debug from Visual Studio 2019?

### ISSUE THREAD

**Issue by jamesmontemagno** [dotnet/net6-mobile-samples#92]
I was able to debug from VS Code, however I am not positive how to launch it from VS 2019.

**Comment by jonathanpeppers** [comment]
You need at least 16.9, but we have a manual step to create a `EnableWorkloadResolver.sentinel` file:

https://github.com/dotnet/net6-mobile-samples#using-ides

I don't think the HelloMaui app will work yet, multi-targeting is still in progress.

**Comment by jamesmontemagno** [comment]
Ahhh gotcha! normal .NET for Android worked, but couldnt' get the .NET MAUI project as the startup. Makes sense.

**Comment by jonathanpeppers** [comment]
You could try deleting `<TargetFrameworks>` plural and putting a single `<TargetFramework>net6.0-android</TargetFramework>` and restart VS to Debug the HelloMaui app.

I was able to use the Remote iOS Simulator this way using `net6.0-ios`, too. You would need to install the iOS workload on Windows, then install *all the things* on the Mac build host.

I think this would help anyone that comes across this, thanks.

**Comment by keithnicholson** [comment]
I'm using VS 2019 16.11 preview and I am unable to build.  I have dotnet  6.0.100-preview.4.21255.9 installed.  Yet I get the error:

The framework 'Microsoft.NETCore.App', version '6.0.0-preview.4.21253.7' was not found.

I also noticed the ReadMe still reflects to 16.10.

**Comment by jonathanpeppers** [comment]
Does your `global.json` match what is installed? Preview 4 final is a different number than what your error message says:

https://github.com/dotnet/maui-samples/blob/d2a8b835c8bd72938ec76c475fe7835802bfb1a4/global.json#L3

**Comment by keithnicholson** [comment]
Good point.  I just cloned the repo night before last and it shows the version I posted.  Let me re-clone and see what happens.
